sprokit/processes/core/video_input_process.cxx

```diff
@@ -195,7 +200,17 @@ ::_step()
       if ( ! d->m_video_traits.capability( kwiver::vital::algo::video_input::HAS_FRAME_TIME ) )
       {
         // create an internal time standard
-        d->m_frame_time = d->m_frame_number * d->m_config_frame_time;
+        double frame_rate = d->m_video_reader->frame_rate();
+        if( ! d->m_video_traits.capability( kwiver::vital::algo::video_input::HAS_FRAME_RATE ) ||
+            frame_rate <= 0.0 || d->m_has_config_frame_time )
+        {
+          d->m_frame_time = d->m_frame_number * d->m_config_frame_time;
```

Comment:
This seem like something the video source should be doing, not the sprokit process.  I would rather have the video source be in charge of figuring out the time for each frame.

Reply:
It looks to me like the frame time configuration is supposed to be a process-level override. If it was happening at the video source level, it would have to be duplicated among all video source implementations.

Also, the old version was doing this in the process class too.

Reply:
okay, I understand now that this is supposed to be an option to override the time from the video source.  I guess that's okay.